### DISCUSSION TITLE
MFA procedure is sticky on step 2

### DISCUSSION THREAD

**Discussion by cridam** [community/community#68582] [New to GitHub]
After scanning and creating account on microsft authenticator, the github procedure go to page , "item 8" in the github FAQ
8) After saving your two-factor recovery codes, click I have saved my recovery codes to enable two-factor authentication for your account.
After clicking over downloading the code, "back" to the page is loosing the context and it is no more possible to continue the procedure. I cannot find any way to complete the procedure.

**Comment by ghost** [comment]
Assuming that you have followed all the steps correctly, including downloading the codes to a specified directory and then clicking 'I have saved my recovery codes,' there shouldn't be any issues. This should enable 2FA on your account.

Things you could try though include pressing CTRL + F5 to refresh the browser cache or logging into GitHub and performing the procedure in a different browser. That's all I can really think of, hope you're able to resolve your issue :+1:
